client/js/lounge.js

```diff
@@ -22,6 +22,7 @@ require("./socket-events");
 const constants = require("./constants");
 const storage = require("./localStorage");
 const utils = require("./utils");
+require("./webpush");
```

Comment:
This is very much side-effecty, I don't like that too much. But rather a nitpick, I guess.